Gopkg.lock

```diff
@@ -7,11 +7,6 @@
   revision = "dbeaa9332f19a944acb5736b4456cfcc02140e29"
   version = "v3.1.0"
 
-[[projects]]
```

Comment:
Good to see this dependency go, but you should not touch the lock file. The way to update dependencies is to remove the lock file and then change the dependency Gopkd.toml. Then you rebuild. Dep will detect the missing lock file and re-create it (verifying and resolving dependencies). The you check in and commit the tomal and lock file. Something like this:

    $ make clean && rm Gopkg.lock
    # remove Idler dependency from Gopkd.toml
    $ make all
    $ git add . && git commit

Reply:
Sure, but I haven't touched the lock file:)

Reply:
Hmm, one way or another you did. It might be that Dep detected that the dependency to _fabric8-jenkins-idler_ is not needed anymore and updated the lock file (a bit odd). Either way, what we should do is removing the dependency in _Gopkg.toml_, deleting _Gopkg.lock_ and then a rebuild. After that we check in the _toml_ as well as the _lock_ file.